### ISSUE TITLE
Error when getting chat messages with nextcloud/axios 2.1.0 

### ISSUE THREAD

**Issue by danxuliu** [nextcloud/spreed#8159]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

Since #8127 the requests to get the chat messages may fail when opening a conversation.

Note that just reloading the page when you have a conversation already open does not trigger the issue; it seems that you need to first open Talk and, then, open a conversation.

## How to test

- Open Talk
- Open a conversation

### Expected result

No errors in the browser console

### Actual result

In the browser console the error `Error happened while getting chat messages. Trying again in  6 TypeError: error.request is undefined` is repeated

**Comment by nickvergessen** [comment]
regression from https://github.com/nextcloud/nextcloud-axios/pull/511 ?

**Comment by danxuliu** [comment]
> regression from [nextcloud/nextcloud-axios#511](https://github.com/nextcloud/nextcloud-axios/pull/511) ?

No, [nextcloud/nextcloud-axios#510](https://github.com/nextcloud/nextcloud-axios/pull/510#issuecomment-1335088531)

**Comment by joshtrichards** [comment]
Fixed in nextcloud-libraries/nextcloud-axios#552 + #8473
However, seems to be brought out a new matter: nextcloud-libraries/nextcloud-axios#579
